woocommerce/woocommerce
# Package Release: Prepare PHP packages for release

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Contributes to https://github.com/woocommerce/woocommerce/issues/44417



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Test out the script on a new PHP package:
```
./tools/package-release/bin/dev prepare @woocommerce/remote-specs-validation --initial-release
```
2. See the script pass and check the resulting changes. The changelog files will be deleted, the changelog updated, and the version bumped in `composer.json`
3. Now reset your work environment and try with a JS package to make sure there are no regressions:
```
./tools/package-release/bin/dev prepare @woocommerce/components
```
4. Make sure the script succeeds, changelog files deleted, changelog updated, and version bumped in `package.json`



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
